### PR TITLE
Issue#1790 general styles

### PR DESCRIPTION
**Pull request by koliapatskal**
#1790 

* Fixed search-bar icons style
* Fixed workshop details
* Fixed form
* The blue color menu has been changed to white.
* Fixed search-bar
* Fixed filters

* Also added a limit on the width of social groups for correct display on mobile phones
image

### CODE REVIEW

File: src/app/shell/personal-cabinet/parent/create-child/child-form/child-form.component.html [hunk 1]
```diff
@@ -3,62 +3,98 @@
   fxLayout="column"
   fxLayoutAlign="center space-between"
   class="step form"
-  [ngClass]="childrenAmount > 1 ? 'step-border' : ''">
+  [ngClass]="childrenAmount > 1 ? 'step-border' : ''"
```
Comment (khrMykhailo) [inline src/app/shell/personal-cabinet/parent/create-child/child-form/child-form.component.html:6]: [class.step-border]="childrenAmount > 1"

**Review by khrMykhailo — APPROVED** [review]

File: src/app/shared/components/filters-list/full-search-bar/full-search-bar.component.scss [hunk 2]
```diff
@@ -30,3 +30,7 @@
   margin: 0 0.5rem 0 1rem;
   color: #AAAAAA;
 }
+
+.search ::ng-deep .mat-icon{
```
Comment (litvinets) [inline src/app/shared/components/filters-list/full-search-bar/full-search-bar.component.scss:34]: space before curly brackets

**Review by litvinets — CHANGES_REQUESTED** [review]
text should be bold 
image
image

File: src/app/shell/personal-cabinet/parent/create-child/child-form/child-form.component.html [hunk 3]
```diff
@@ -117,17 +174,25 @@
...
-    [maxCharachters]="validationConstants.INPUT_LENGTH_30">
+    [maxCharachters]="validationConstants.INPUT_LENGTH_30"
+  >
   </app-validation-hint>
 
-  <div fxLayout="row" fxLayoutAlign="center" *ngIf="childrenAmount > 1" class="addChild">
+  <div
+    fxLayout="row"
+    fxLayoutAlign="center"
+    *ngIf="childrenAmount > 1"
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/parent/create-child/child-form/child-form.component.html:190]: *ngIf="!!childrenAmount"

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**